src/Kovaleva.HomeWork-4/Program.cs

```diff
@@ -9,11 +9,38 @@ static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
         }
-    }
 
-    private static List<Task> InitTaskList()
-    { var TaskList = new List<Task>();
+        private static List<Task> InitTaskList()
+        {
+            var TaskList = new List<Task>();
```

Comment:
let's use common naming convention for .net where variable names start from lower case character